app/views/courses/_course_lessons.html.erb

```diff
@@ -1,80 +1,90 @@
 <% modules_in_order(course).each do |course_module| %>
-  <details class="border-line-colour mx-4 mb-6 rounded border">
-    <summary class="flex items-center justify-between p-6 cursor-pointer">
-      <h2 class="text-black0 text-base font-semibold">
-        <% if policy(course_module).show? %>
-          <%= link_to course_module.title, course_module_path(course, course_module), class: "nav-link" %>
-        <% else %>
-          <%= course_module.title %>
-        <% end %>
-        <span class="text-primary pl-5 text-sm font-medium">(<%= lessons_count(course_module) %>)</span>
-      </h2>
-      <div class="flex gap-4 items-center">
-        <% if policy(@course).edit? %>
-          <%= link_to edit_course_module_path(course, course_module) do %>
-            <%= render 'shared/components/button_default_small', label: 'Edit', icon_name: 'icon-edit' %>
+  <div data-controller="details">
+    <!-- TODO Disabled temporarily -->
```

Comment:
Just add a single line todo indicating what need to be done here. Remove the line on 8

Reply:
removed the extra line